packages/gatsby-plugin-graphql-config/package.json

```diff
@@ -1,18 +1,23 @@
 {
   "name": "gatsby-plugin-graphql-config",
   "description": "Gatsby plugin to write out a graphql-config with develop process endpoint configured",
-  "version": "1.0.0",
+  "version": "0.0.0",
   "author": "Rikki Schulte <rikki.schulte@gmail.com>",
   "bugs": {
     "url": "https://github.com/gatsbyjs/gatsby/issues"
   },
+  "dependencies": {
+    "fs-extra": "^9.0.1",
+    "graphql": "^15.3.0"
```

Comment:
I think we should just change imports in the code to `import x from "gatsby/graphql"` and don't require graphql, otherwise, we can end up with multiple versions of graphql in the project (Gatsby is still on `14.x`).

Reply:
thanks! :)

Reply:
Let me check real quick if it still works this way :)

Reply:
Works OK for me 👍

Reply:
Did it work? @vladar ?

Reply:
@wardpeet Yes! Packed and tested on a fresh blog starter project.